### PR TITLE
[fcos] cherrypick UPI fixes

### PR DESCRIPTION
**Pull request by vrutkovs**
This should resolve issues on vSphere UPI test.

* cherry-pick two commits to improve stability of bootstrap:
  https://github.com/openshift/installer/pull/3031 and https://github.com/openshift/installer/pull/3007
* Use default `crun` to avoid confusing podman
* Disable zincati service on bootstrap
  TODO: do the same on masters / workers
* prepull podman images

### CODE REVIEW

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -59,17 +59,16 @@ if [ ! -f .pivot-done ]; then
   chmod +x /usr/local/bin/machine-config-daemon
   restorecon /usr/local/bin/machine-config-daemon
 
+	# Disable zincati
+	systemctl disable zincati.service
```
Comment (LorbusChris) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: indentation

**Comment by vrutkovs** [comment]
vSphere bootstrapped but authentication operator got stuck (Route53 issues?): https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_installer/3013/pull-ci-openshift-installer-fcos-e2e-vsphere/399

/retest

**Pull request merged**